File: INICIANTE/JAVASCRIPT/1035.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var prompt = function(texto) { return lines.shift(); };

var [A, B, C, D] = prompt("A").split(" ").map(Number)

if(B > C && D > A && C + D > A + B && C + D > 0 && A + B > 0 && A % 2 == 0){
console.log("Valores aceitos")
    
}
else{
    console.log("Valores nao aceitos")
}

File: INICIANTE/JAVA/1035.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        int A, B, C, D;
        Scanner sc = new Scanner(System.in);
        A = sc.nextInt();
        B = sc.nextInt();
        C = sc.nextInt();
        D = sc.nextInt();
        if(B > C && D > A && C + D > A + B && C + D > 0 && A + B > 0 && A % 2 == 0){
            System.out.print("Valores aceitos" + "\n");
                
        }else{
            System.out.print("Valores nao aceitos" + "\n");
        }
    }
} 
